index.js

```diff
@@ -2,10 +2,14 @@ const express = require('express');
 const app = express();
 const server = app.listen(7777);
 
-app.get('*', (req, res) => {
+app.get('/', (req, res) => {
     res.send('API working fine');
 });
 
+app.get('/second-route', (req, res) => {
+    res.send('second route working fine');
```

Comment:
i think this should say `res.send('second route fine')`

Reply:
okay